src/lib.rs

```diff
@@ -377,16 +377,22 @@ impl Fann {
 
     /// Read a neural network from a file.
     pub fn from_file<P: AsRef<Path>>(path: P) -> FannResult<Fann> {
-        let filename = try!(to_filename(path));
+        let filename = to_filename(path)?;
         unsafe { Fann::from_raw(fann_create_from_file(filename.as_ptr())) }
     }
 
+    /// Create a deep copy of a neural network.
+    /// The `Clone` trait is intentionally not implemented, because this operation might fail.
```

Comment:
```suggestion
    /// Create a deep copy of a neural network.
    ///
    /// The `Clone` trait is intentionally not implemented, because this operation might fail.
```

(Probably looks better in the generated HTML.)